README.md

```diff
@@ -1,4 +1,47 @@
+[![Build Status](https://travis-ci.org/sesam-community/azure-storage.svg?branch=master)](https://travis-ci.org/sesam-community/azure-storage)
+
+
 # azure-storage
-Sesam datasource that reads files from Azure Storage
+Can be used to
+ * reads files from Azure Storage(Blob/Files)
+ * upload files to Azure Storage(Files)
 
-[![Build Status](https://travis-ci.org/sesam-community/azure-storage.svg?branch=master)](https://travis-ci.org/sesam-community/azure-storage)
+ ### Environment Parameters
+
+ | CONFIG_NAME        | DESCRIPTION           | IS_REQUIRED  |DEFAULT_VALUE|
+ | -------------------|---------------------|:------------:|:-----------:|
+ | ACCOUNT_NAME | Azure Storage account name. | no, basic auth alternatively | n/a |
+ | ACCOUNT_KEY |  Azure Storage account key. | no, basic auth alternatively | n/a |
+ | SAS_PARAMS | dict for the sas_token generation parameters | no | check the code |
+
+
+ ### Query Parameters
+
+ ##### POST REQUEST:
+
+ | CONFIG_NAME        | DESCRIPTION           | IS_REQUIRED  |DEFAULT_VALUE|
+ | -------------------|---------------------|:------------:|:-----------:|
+ | start_timedelta | time offset for the start of sas_token validity. <numeric_value> followed by M, H and D for minutes, hours and days, respectively. | no | 0M (immediate start) |
+ | expiry_timedelta | time offset for the expiry of sas_token validity. <numeric_value> followed by M, H and D for minutes, hours and days, respectively.  | no | 12H |
+
+
+
+ ### An example of system config:
+
+ system:
+ ```json
+ {
+   "_id": "my_azure_storage",
+   "type": "system:microservice",
+   "connect_timeout": 60,
+   "docker": {
+     "environment": {
+       "ACCOUNT_NAME":"myaccount",
+       "ACCOUNT_KEY":"myaccount-key"
+     },
+     "image": "mydockeruser/azure-storage:v1.0",
```

Comment:
Is the tag correct? The community builds currenly only support lastest as far as I know. 

The example should use the sesamcommunity user in the example.

Reply:
community builds support other tags.